api/controllers/login.go

```diff
@@ -73,8 +101,24 @@ func (ctrl *Controller) OAuthCallbackHandler(c *gin.Context) {
 		return
 	}
 
-	c.JSON(http.StatusOK, gin.H{
-		"access_token": userAuthToken.Token,
-		"expires_at":   userAuthToken.ExpiresAt,
-	})
+	c.Header("Set-Cookie", fmt.Sprintf("Access-Token=%s; Expires=%s; HttpOnly; Secure; SameSite=Lax; Path=/", userAuthToken.Token, userAuthToken.ExpiresAt.Format(http.TimeFormat)))
```

Comment:
Is "Access Token" standard, or should we make it look salz-specific? e.g. `Salz-Access-Token`?

Reply:
its a nitpick though, doesn't really matter to me